variables.tf

```diff
@@ -71,8 +71,8 @@ variable "resources_driver" {
   type        = map(any)
   default = {
     requests = {
-      cpu    = "200m"
-      memory = "200Mi"
+      cpu    = "50m"
+      memory = "100Mi"
```

Comment:
Make memory request identical to limit always.

Reply:
Ok i can update that, i just followed the below resource requests. 

https://github.com/kubernetes-sigs/secrets-store-csi-driver/blob/main/charts/secrets-store-csi-driver/values.yaml

Reply:
I will update and push the changes again